packages/mdl2-theme/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@uifabric/mdl2-theme",
+  "version": "0.1.0",
```

Comment:
Should it just start at 7.0.0?

Reply:
We wanted to keep it at 0 major to try and reduce the desire of using it. The main purpose of this package is to facilitate ODSP repos transition to 7.0

Reply:
It's probably okay to release as it is but note that < 1 releases are always very messy (caret sign has different meaning). Also if we were to release Fabric 8 eventually, we likely will have to bump this to 8.

It is not a blocking comment. Based on what I looked at, I think what you have shouldn't cause any immediate problems. Was just asking the question :) I suspect it is quite harmless to bump to 7 from the get-go and would be more consistent with the rest of the packages, but don't have strong opinions.

(also, it feels like this package will be universally useful right for anyone who wants to upgrade to Fabric 7 but stick with mdl2 look?)